pages/index.php

```diff
@@ -8,7 +8,7 @@
 // https://github.com/redaxo/redaxo/pull/5664
 
 echo '
-<script type="text/javascript">
+<script type="text/javascript" nonce="'.rex_response::getNonce().'">
```

Comment:
ggf. macht es auch sinn die javascripts in eine externe datei zu verlagern und via `<script src ..>` zu laden.. dann müsste es auch ohne `nonce` gehen, bzw. der core diese automatisch generieren, oder?

Reply:
Im Backend haben wir derzeit auch an jedem `<script src="..">` das Attribute nonce dran. Das kommt automatisch, sobald man `rex_view::addJSFile()` nutzt.

Reply:
jop das meinte ich. wenn wir den code in externe dateien verlagern und `rex_view::addJSFile()` verwenden müsste das problem auch gelöst sein?